Tests/MapboxNavigationTests/UserHaloCourseViewSnapshotTests.swift

```diff
@@ -38,14 +38,14 @@ class UserHaloCourseViewSnapshotTests: TestCase {
         
         XCTAssertEqual(userHaloCourseView.layer.sublayers?.count, 1, "UserHaloCourseView should contain only one layer.")
         
-        assertImageSnapshot(matching: userHaloCourseView, as: .image(precision: 1.00))
+        assertImageSnapshot(matching: userHaloCourseView, as: .image(precision: 0.95))
```

Comment:
To make consistent with other tests, all other screenshot tests have 0.95 precision